automatic/paint.net/update.ps1

```diff
@@ -19,11 +22,16 @@ function global:au_BeforeUpdate {
 
 function global:au_GetLatest {
     $LatestRelease = Get-GitHubRelease paintdotnet release
+    Get-RemoteFiles -Purge -NoSuffix
+    Set-Alias 7z $Env:chocolateyInstall\tools\7z.exe e tools\*.zip -otools *.msi -r -y
+  rm tools\*.zip -ea 0
```

Comment:
These three lines is not needed, and would cause issues in the long run.

This is already handled in the `au_BeforeUpdate` function, so please remove these lines.

Reply:
Have been removed from the au_GetLatest